main/FatesInventoryInitMod.F90

```diff
@@ -69,6 +69,9 @@ module FatesInventoryInitMod
                                                            ! defined in model memory and a physical
                                                            ! site listed in the file
 
+   logical, parameter :: do_inventory_out = .true.
```

Comment:
is the plan to test the PR with this `do_inventory_out` switch set as true to confirm correct behavior, but then change it back to false before merging?

Reply:
It should be changed back to false before pulling.  We you able to use this output, did it work correctly for you @ckoven ?

Reply:
yes it is working correctly for me at present